source/2020-05-01-the-ember-times-issue-146.md

```diff
@@ -46,14 +47,13 @@ Interested in writing a post about an a11y topic such as screenreaders, WCAG, or
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Call for contributors: Shipping the Ember website(s) redesign 🎨🐹](https://twitter.com/melaniersumner/status/1254497875062673411)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Early last year the community came together to [discuss the future of the design strategy of official Ember sites](https://twitter.com/emberjs/status/1105157291806883840). After plenty of time and work investments from individual contributors, companies and the Ember Core team, the redesign for the [original RFC](https://emberjs.github.io/rfcs/0425-website-redesign.html) had [shipped successfully](https://twitter.com/emberjs/status/1230912205631213569) for the main site of the project: [emberjs.com](https://emberjs.com)!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+But many other official resources within the Ember community are looking forward to get a new look & feel as well - and you can help with that! A [call for contributors](https://twitter.com/melaniersumner/status/1254497875062673411?s=21) is looking for those who have experience with JavaScript, HTML and CSS and who want to help shipping the website redesign to e.g. Ember's API Docs or the CLI Guides.
```

Comment:
I thought we could provide the examples of sites that need a redesign in the first sentence.

I haven't been following the redesign plan closely. I added the Blog as the first example because I vaguely remember that being the priority right now? (Maybe?)
```suggestion
Many other official resources for the Ember community, such as the [Blog](https://blog.emberjs.com/), [API Docs](https://api.emberjs.com/), and [CLI Guides](https://cli.emberjs.com/), are looking forward to get a new look & feel as well - and you can help with that! A [call for contributors](https://twitter.com/melaniersumner/status/1254497875062673411?s=21) is looking for those who have experience with JavaScript, HTML and CSS and who want to help shipping the website redesign.
```

Reply:
Sounds awesome, thank you for the suggestion!